num2words/currency.py

```diff
@@ -1,39 +1,31 @@
 from __future__ import division
 
-from decimal import Decimal
+from decimal import ROUND_HALF_UP, Decimal
 
 
-def parse_currency_parts(value):
+def parse_currency_parts(value, is_int_with_cents=True):
```

Comment:
I don't think this change is necessary, you could check if the integer part returned is '0' and use only the cents part in your code instead of change this function params. For the rest of the changes in this function, it looks OK to remove the `elif isinstance(value, Decimal)` and convert the value to a Decimal value directly.

Reply:
Sorry, I don't understand this part.
"you could check if the integer part returned is '0' and use only the cents part in your code instead of change this function params"

Without `is_int_with_cents` you're assuming all int is dollar with 2 decimal cent.

From my understand at least in Thailand, 
when we type 123 USD we means 123 USD not 1 USD and 23 cents.

Could you please elaborate.

Thanks.